.github/workflows/update_appraisals.yml

```diff
@@ -1,7 +1,6 @@
 ---
 name: Update Appraisals
 on:
-  push: ~
```

Comment:
This was running when we pushed to the repo, which then caused Unit Tests to not run on my branch due to how Github Workflows work, where they cannot get triggered from other GithubWorkflows that use `git config`

Reply:
Ah yeah that sucks now that you mention it. The weekly build is going to silently cause us problems if we update the appraisals without running the tests. We should change the job so that we can only commit/push if a `bundle exec appraisal (rspec|rake)` succeeds.

I'll update the [Gem CI Best Practices](https://invoca.atlassian.net/wiki/spaces/DEV/pages/60055388217/Gem+CI+Best+Practices) example once we decide on how this should actually look

Reply:
Hmm, this `Update Appraisals` step is interesting. Its doing two things:
1. Ensures Appraisal Gemfiles are kept up to date with any changes to the `Gemfile`.
2. Ensure we create gemfiles for new Rails minor versions (probably less important?).

I'm curious if we should do something here.

Reply:
I don't see any issue with getting two for one here. In either scenario, it's the same command and we want to run the test suite against the change before pushing. If you don't want (2) to happen then you just need to update the Appraisals file right?

Reply:
The current `Update Appraisal` schedule doesn't even work:
https://github.com/Invoca/aggregate/actions/runs/9827523944/job/27130301493

Reply:
Hmm most of our gems don't require PRs since it's common practice to push release commits onto mainline. I'd just expect that we'd need to run tests and manually write the "build success" status since the action won't trigger unit tests automatically

Reply:
I think we need to completely rethink the Update Appraisal step in its own ticket, to properly think through all of these edge cases.

Reply:
I'm removing the UpdateAppraisals workflow as it is broken in its current state